wire-ios-canvas/WireCanvas.xcodeproj/project.pbxproj

```diff
@@ -381,6 +381,7 @@
 /* Begin PBXShellScriptBuildPhase section */
 		66B66B63275A324C0020CF55 /* Run Swiftlint */ = {
 			isa = PBXShellScriptBuildPhase;
+			alwaysOutOfDate = 1;
```

Comment:
This tells Xcode that we intend to run it on every build. Otherwise Xcode assumes, that it has to find out, if running this is phase is required. But since we didn't define dependencies for the SwiftGen phase, Xcode shows the warning `Run script build phase '...' will be run during every build because it does not specify any outputs...`.